components/src/core/components/Input/Select/SelectInputButton.vue

```diff
@@ -27,26 +28,59 @@
       <template v-if="buttonData.iconImageSrc" v-slot:icon>
         <img :src="buttonData.iconImageSrc" />
       </template>
-      <template #label>
+      <template v-if="modelValue && modelValue.id > -1" #label>
```

Comment:
Do not add -1 and id attribute to the components. Now it is binding to the data that it passes

Reply:
Fixed; Now the component doesn't have -1 checking. just check modalValue is available instead.
SVN vue app fixes
https://websvn.orangehrm.com/orangehrm?op=comp&compare[]=/enterprise/trunk/@172652&compare[]=/enterprise/trunk/@172653

Reply:
Fixed; Now the component doesn't have -1 checking. just check modalValue is available instead.
SVN vue app fixes
https://websvn.orangehrm.com/orangehrm?op=comp&compare[]=/enterprise/trunk/@172652&compare[]=/enterprise/trunk/@172653